### ISSUE TITLE
Taint Analysis: `whereColumn()`/`orWhereColumn()` carry no SQL sink (false negative)

### ISSUE THREAD

**Issue by alies-dev** [psalm/laravel-psalm-plugin#1303]
`whereColumn()` / `orWhereColumn()` are not stubbed, so none of their parameters carry a SQL taint sink. This call is silent today:

```php
$builder->whereColumn((string) $request->input('c'), '=', 'other_column'); // no TaintedSql
```

**Why all three positions are dangerous** (verified against `Illuminate\Database\Query\Grammars\Grammar::whereColumn()`):

```php
return $this->wrap($where['first']).' '.$where['operator'].' '.$this->wrap($where['second']);
```

`$first`, `$operator`, and `$second` all concatenate into raw SQL. Nothing is bound.

**Divergence from `where()`:** `where()` degrades an invalid operator into a bound value. `whereColumn()` instead swaps it into `$second` (`[$second, $operator] = [$operator, '=']`), which is `wrap()`ed as an identifier, still raw. A tainted operator is dangerous either way.

**Array form:** `whereColumn([[$first, $op, $second]])` dispatches through `addArrayOfWheres(..., 'whereColumn')`; every dispatched position is an identifier or raw operator, none bound. `whereColumn` is deliberately absent from `WhereColumnTaintHandler::WHERE_MAP_METHODS` (that handler strips sinks only from PDO-bound positions of the `where()` family) and must stay absent.

**Plan:**

- Add `whereColumn` + `orWhereColumn` to `stubs/common/Database/Query/Builder.phpstub` with `@psalm-taint-sink sql` on `$first`, `$operator`, `$second`.
- The Eloquent side resolves through `@mixin \Illuminate\Database\Query\Builder`, so one stub covers both builders (same mechanism as the `whereLike` family, #1302).
- phpt coverage: each sink position, `orWhereColumn`, Eloquent chain, and a regression pin that the array form keeps flagging (guards the `WHERE_MAP_METHODS` allowlist).

Probe note: static `Model::whereColumn(...)` currently resolves through the mixin without `UndefinedMagicMethod`; it is silent purely because the method is unstubbed, so the stub fixes the static path as well.
